packages/pirateship/src/components/PSScreenWrapper.tsx

```diff
@@ -30,13 +30,15 @@ const styles = StyleSheet.create({
   }
 });
 
+
 export interface PSScreenWrapperProps {
   style?: StyleProp<ViewStyle>;
 
   hideGlobalBanner?: boolean;
   overrideGlobalBanner?: PSGlobalBannerSlotItem;
 
   needInSafeArea?: boolean;
+  navigator?: Navigator;
```

Comment:
I don't think this is necessary because we're defining navigator on line 46. If you're getting a tsc error it's probably coming from somewhere else.